onnxruntime/core/providers/openvino/openvino_mo/openvino_mo.py

```diff
@@ -377,6 +406,106 @@ def driver_R1(onnx_modelproto_bytes, precision: str, output_model_name: str, out
 
     return weights, xml_string
 
+def driver_2019_R2(onnx_modelproto_bytes, precision: str, output_model_name: str, output_dir: str):
+    
+    print('Entered driver_2019_R2')
+    try:
+        model_proto = onnx.load_from_string(bytes(onnx_modelproto_bytes))
+    except Exception as e:
+        print("[python] onnx exception: ", str(e))
+
+    model_graph = model_proto.graph  # pylint: disable=no-member
+    log.debug("Number of nodes in graph_def: {}".format(len(model_graph.node)))
+    log.debug("Number of all input ports (not true inputs) in graph_def: {}".format(len(model_graph.input)))
+    log.debug("Number of initializers in graph_def: {}".format(len(model_graph.initializer)))
+    log.debug("Number of real inputs in graph_def: {}".format(len(model_graph.input) - len(model_graph.initializer)))
```

Comment:
Note this [doesn't hold](https://github.com/onnx/onnx/commit/dc75285d4a1cff9618400164dfdb26c5a1bab70a) for IR version >= 4